Translations/translation_ru.json

```diff
@@ -24,7 +24,7 @@
 		"ResetOKMessage": "Сброс OK",
 		"YourGainMessage": "Прирост:",
 		"SettingsResetMessage": "Настройки сброшены!",
-		"NoAccelerometerMessage": "No accelerometer\ndetected!",
+		"NoAccelerometerMessage": "Нет акселерометра\не определен!",
```

Comment:
This is probably not what you want  the \n means go go to a new line

Reply:
Thanks I fixed it.